src/i18n/de/branding.json

```diff
@@ -0,0 +1,3 @@
+{
+  "powered by": "powered by"
```

Comment:
The proper German translation (in terms of actual usage in the wild) is indeed... "powered by" ;-)